includes/definitions/discovery/ptp500.yaml

```diff
@@ -0,0 +1,47 @@
+mib: CAMBIUM-PTP500-V2-MIB
+modules:
+    sensors:
+        state:
+            data:
+                -
+                    oid: transmitModulationMode
+                    num_oid: .1.3.6.1.4.1.17713.5.12.9.
+                    index: 0
```

Comment:
You should use something better for the index in all of these. `'transmitModulationMode.{{ $index }}'` is what we would normally do.